RadixWallet/Features/AssetsFeature/Components/HelperViews/EmptyAssetListView.swift

```diff
@@ -3,40 +3,60 @@ import SwiftUI
 
 // MARK: - EmptyAssetListView
 struct EmptyAssetListView: View {
-	struct ViewState {
+	struct ViewState: Sendable, Equatable {
 		let imageAsset: ImageAsset
 		let description: String
+		let glossaryItem: OverlayWindowClient.GlossaryItem
+		let glossaryLabel: String
 	}
 
 	let viewState: ViewState
 
+	init(_ viewState: ViewState) {
+		self.viewState = viewState
+	}
+
 	var body: some View {
 		VStack(spacing: .large2) {
 			Image(asset: viewState.imageAsset)
 			Text(viewState.description)
 				.foregroundColor(.app.gray1)
 				.textStyle(.sectionHeader)
+
+			InfoButton(viewState.glossaryItem, label: viewState.glossaryLabel)
 		}
 		.padding(.top, .large2)
 		.centered
 		.listRowBackground(Color.clear)
 	}
 }
 
-extension EmptyAssetListView {
-	static var fungibleResources: Self {
-		.init(viewState: .init(imageAsset: AssetResource.fungibleTokens, description: L10n.AssetDetails.TokenDetails.noTokens))
-	}
+extension EmptyAssetListView.ViewState {
+	static let fungibleResources = Self(
+		imageAsset: AssetResource.fungibleTokens,
+		description: L10n.AssetDetails.TokenDetails.noTokens,
+		glossaryItem: .tokens,
```

Comment:
It feels wrong to have to pass two values: `glossaryItem` **and** `glossaryLabel`. I think the "label" should be a computed property on the enum itself, just like you do with `OverlayWindowClient.GlossaryItem`'s computed property `string`. 

Maybe `GlossaryItem` should not be an enum but rather an "open enum"? i.e. a struct and have `static let tokens = Self(title: ..., image: ..., body: ...)` ?

Reply:
It's different though. The label depends on the context where the button is shown, it's the text shown on the button itself, but the `string` property is what is shown on the sheet, which corresponds directly to the glossaryItem. 

One reason to keep it an enum is that we have a fixed set of "glossaryAnchors" in the markdown.

Reply:
In this particular case, the label is analogous to the `description` property, and indeed they are already organized in the same L10n folder.

Reply:
ok makes sense